.env.example

```
@@ -236,6 +236,10 @@ BACKUP_ON_DELETE=true
 MEDIA_BROWSER_ENABLED=false
 
 
+# Whether to allow embedding songs, albums, artists, and playlists on external sites.
+EMBED_ENABLED=true
+
+
```

dotenv-linter (4.0.0)

[warning] [ExtraBlankLine] Extra blank line detected

(ExtraBlankLine)